shared/video-sdk/develop/real-time-transcription/project-implementation/android.mdx

```diff
@@ -334,12 +335,12 @@ Take the following steps to start, stop and query a speech-to-text task in your
             public void onResponse(Call call, final Response response) throws IOException {
                 String result = response.body().string();
                 if (response.isSuccessful()) {
-                    showMessage("Speech-to-text task stopped");
-                    speechToTextTaskId = "";
-                    isSpeechToTextRunning = false;
-                    runOnUiThread(() -> btnSpeechToText.setText("Start speech-to-text"));
+                    showMessage("Real-time transcription task stopped");
+                    realTimeTranscriptionTaskId = "";
+                    isRealTimeTranscriptionRunning = false;
+                    runOnUiThread(() -> btnRealTimeTranscription.setText("Start real-time transcription"));
                 } else {
-                    showMessage("Exception stopping speech-to-text task: " + result);
+                    showMessage("Exception stopping real-time transcription: " + result);
```

Comment:
I suppose it is the task we are dealing as above message shows "showMessage("Real-time transcription task stopped");"

So we should keep ""task" her is message as in original doc .
```
} else {
                    showMessage("Exception stopping real-time transcription task: " + result);